plasma_framework/test/src/vaults/Erc20Vault.test.js

```diff
@@ -117,6 +119,21 @@ contract('Erc20Vault', (accounts) => {
                 'Must have only one output',
             );
         });
+
+        it('should not store a deposit in a newly registered vault', async () => {
```

Comment:
I think it is better to test the feature in `BlockController` instead of `Vault`.

 I think what we want is: `PlasmaFramework can protect user from operator registering bad vaults` 
and not: `Our new vault has the feature of not being able to depositBlock before certain time`

Also, it reduces the tests needed.